src/govuk/components/summary-list/_index.scss

```diff
@@ -166,4 +166,72 @@
       }
     }
   }
+
+  // Additional block for the summary card
+  //
+  // We ship this with the summary list instead of making it it's own component
+  // because it's only interacted with via the summary list and it avoids awkwardness
+  // with trying to document a "private" component that's controlled via the
+  // component it's wrapping
+  .govuk-summary-card {
+    @include govuk-responsive-margin(6, "bottom");
+    border: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-summary-card__title-wrapper {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: govuk-spacing(3) govuk-spacing(4);
```

Comment:
@christopherthomasdesign do we want to align the left-edge with the summary card contents on mobile?

<img width="439" alt="211529187-364d3614-dd3b-4824-858b-746825c16b24" src="https://user-images.githubusercontent.com/121939/211764376-d3aa74e4-dea1-408b-a404-2ea90768074c.png">

Reply:
Yes we definitely do @36degrees, this is an oversight on my part!

Reply:
I'm trying to figure out where this gap is coming from but I'm a bit stumped. May confer with the channel later today.

Reply:
It's a difference in padding – `govuk-summary-card__title-wrapper` has `govuk-spacing(4)` for horizontal spacing at all breakpoints, but `govuk-summary-card__content` has `govuk-spacing(3)` on mobile and `govuk-spacing(4)` on tablet and above.

So `govuk-summary-card__title-wrapper` needs to be updated to match `govuk-summary-card__content`:

```suggestion
    padding: govuk-spacing(3);
    
    @include govuk-media-query($from: "tablet") {
      padding: govuk-spacing(3) govuk-spacing(4);
    }
```

Reply:
Doh! Cheers for point that out. Gonna fix.